test/unit/gateways/cyber_source_test.rb

```diff
@@ -451,22 +451,34 @@ def test_credit_includes_mdd_fields
     end.respond_with(successful_card_credit_response)
   end
 
+  def test_successful_void_purchase_request
+    authorization = '1000;1842651133440156177166;AP4JY+Or4xRonEAOERAyMzQzOTEzMEM0MFZaNUZCBgDH3fgJ8AEGAMfd+AnwAwzRpAAA7RT/;purchase;100;USD;'
```

Comment:
Do you think that you could change this to `purchase` to reflect what type of action it is? Same for the capture on line 465.

Reply:
Yeah, that makes more sense. Thanks for the review @Carrigan!